docs/graphcache/cache-updates.md

```diff
@@ -543,8 +543,43 @@ variables using our query document definitions, which means that our API will ne
 variables other than the ones we've defined.
 
 However, we're able to pass additional variables to our mutation, e.g. `{ extra }`, and since
-`$extra` isn't defined it will be filtered once the mutation is sent to the API. An optimistic
-mutation however will still be able to access this variable.
+`$extra` isn't defined it will be filtered once the mutation is sent to the API.
+
+```jsx
+const MyComponent = () => {
+  const [_, executeMutation] = useMutation(UpdateTodo);
+
+  return (
+    <button
+      type="button"
+      onClick={() =>
+        executeMutation({
+          id: 'anId',
+          text: 'New value',
+          // Let's pass an extra variable
+          extra: 'An extra value',
+        })
+      }
+    >
+      Update todo
+    </button>
+  );
+};
+```
```

Comment:
I'm not really sure, we want to have React specific code in the **Graphcache** documentation. 
I don't know how to show the mutation in a framework/library agnostic way. Do you have an idea?

Reply:
I would opt to simply not have a code sample in here, sorry! 😅
We generally try to prefer to show no specific bindings in any part of the documentation that is kept generic. While this is sometimes ignored in some "Basics" documentation or when it's absolutely necessary. From experience, I don't think a code sample here helps much.

I've seen that code samples don't help discoverability. They simply take up more space. While that seems like it's more likely for someone skimming the docs to discover it, we can't have all parts of the documentation compete for space and attention.

So, I'd prefer to leave this as is. Sorry! ❤️

Reply:
I understand, I don't think this code is that helpful :)
I remove this one. 
But I think the other one is useful